src/genpy/generator.py

```diff
@@ -896,6 +897,7 @@ def srv_generator(msg_context, spec, search_path):
     genmsg.msg_loader.load_depends(msg_context, spec, search_path)
     md5 = genmsg.compute_md5(msg_context, spec)
 
+    yield '# This Python file uses the following encoding: utf-8'
```

Comment:
This is unrelated to the pull request, but should this file also have a note about being auto-generated, as above:

https://github.com/ros/genpy/pull/41/files#diff-cda8e9d50594619a9ea61bdd3216b820R750